kale/task.py

```diff
@@ -89,15 +89,15 @@ def _get_task_id(cls, *args, **kwargs):
         return '%s_uuid_%s' % (cls.__name__, uuid.uuid1())
 
     @classmethod
-    def publish(cls, app_data, *args, **kwargs):
+    def publish(cls, app_data, *args, delay_sec=None, **kwargs):
```

Comment:
`*args` needs to be after all defined args and kwargs.

Reply:
@sabw8217 Thanks for pointing this out. We assumed that we only needed this code to work in python 3, where we are able to define "keyword-only" arguments like this, but looking at the setup file and the travis config, seems like we need this code to run in python 2.7 also.

I changed this instead to not alter the function signature, but instead try to dig `delay_sec` out of the kwargs.